practice_exercises.rb

```diff
@@ -0,0 +1,68 @@
+# Time Complexity: O(n)
```

Comment:
Just a note... This file should be in the `lib` folder.  Otherwise running the tests is a pain for me.